src/sql/workbench/contrib/query/browser/queryActions.ts

```diff
@@ -198,7 +200,8 @@ export class RunQueryAction extends QueryTaskbarAction {
 		editor: QueryEditor,
 		@IQueryModelService protected readonly queryModelService: IQueryModelService,
 		@IConnectionManagementService connectionManagementService: IConnectionManagementService,
-		@ICommandService private readonly commandService?: ICommandService
+		@ICommandService private readonly commandService?: ICommandService,
+		@IAdsTelemetryService private readonly telemetryService?: IAdsTelemetryService,
```

Comment:
You should be passing in a [NullAdsTelemetryService](https://github.com/Microsoft/azuredatastudio/blob/1c9ba64ee0ad79bbcdfb704038ba4cd7fd8bc986/src/sql/platform/telemetry/common/adsTelemetryService.ts#L217) instead of having this be undefined, services should never be undefined at runtime so having them be undefined in tests is best avoided. 

(the command service should also have been made like that, but it's fine to leave it like it is if you want since that was already set up like that)